src/gleam/io.gleam

```diff
@@ -73,24 +73,39 @@ if javascript {
 /// ```
 ///
 pub fn debug(term: anything) -> anything {
-  debug_print(term)
+  do_debug(term)
   term
 }
 
 if erlang {
-  fn debug_print(term: anything) -> DoNotLeak {
-    erlang_fwrite("~tp\n", [term])
-  }
+  external fn do_debug(term: anything) -> Nil =
+    "gleam_stdlib" "debug"
 }
 
 if javascript {
-  external fn debug_print(anything) -> Nil =
+  external fn do_debug(term: anything) -> Nil =
     "../gleam_stdlib.mjs" "debug"
 }
 
+/// Prints a value to standard output (stdout) yielding Erlang or JavaScript syntax.
+///
+pub fn debug_raw(term: anything) -> anything {
```

Comment:
Let's just have the regular debug function and get rid of this one. 👍

Reply:
They could be useful for people transitioning between Gleam and Erlang/JavaScript (interop), so they know what values to expect on the other side?